fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/MetricUnionAvailability.java

```diff
@@ -81,28 +75,15 @@
      *
      * @param availabilities  A set of <tt>Availabilities</tt> whose Dimension schemas are (typically) the same and
      * the Metric columns are unique(i.e. no overlap) on every availability
-     * @param columns  The set of all configured columns, including dimension columns, that metric union availability
+     * @param availabilitiesToMetricNames  A map of all availabilities to set of metric names
      * will respond with
      */
-    public MetricUnionAvailability(@NotNull Set<Availability> availabilities, @NotNull Set<Column> columns) {
```

Comment:
We should create a new constructor and mark this old one as "deprecated" for backward compatibility.

Reply:
not critical, since no one is using it right now and that it simply does not work

Reply:
More importantly, this is currently broken and we're better off giving a compile time failure because the method is missing than, if they are using it, a run time failure later in the process.